src/commands/exportApi.ts

```diff
@@ -1,32 +1,25 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
 import { IActionContext } from "@microsoft/vscode-azext-utils";
 import * as fs from "fs-extra";
+
 import * as path from "path";
 import * as vscode from "vscode";
-import { ApiCenterService } from "../azure/ApiCenter/ApiCenterService";
 import { ApiSpecExportResultFormat } from "../azure/ApiCenter/contracts";
-import { TelemetryClient } from '../common/telemetryClient';
+import { ApiCenterVersionDefinitionManagement } from "../azure/ApiCenterDefines/ApiCenterDefinition";
 import { ext } from "../extensionVariables";
 import { ApiVersionDefinitionTreeItem } from "../tree/ApiVersionDefinitionTreeItem";
 import { createTemporaryFolder } from "../utils/fsUtil";
+import { GeneralUtils } from "../utils/generalUtils";
 export namespace ExportAPI {
     export async function exportApi(
         context: IActionContext,
         node?: ApiVersionDefinitionTreeItem): Promise<void> {
         if (!node) {
-            node = await ext.treeDataProvider.showTreeItemPicker<ApiVersionDefinitionTreeItem>(new RegExp(`${ApiVersionDefinitionTreeItem.contextValue}*`), context);
+            node = await ext.treeDataProvider.showTreeItemPicker<ApiVersionDefinitionTreeItem>(new RegExp(`${ApiCenterVersionDefinitionManagement.contextValue}*`), context);
```

Comment:
So from Command Palette, user could only select a tree item in Management Plane View?
When user only have Data Plane view, the Command Palette would be useless to user?

Reply:
updated